tools/msgfplus/msgfplus.xml

```diff
@@ -56,6 +55,13 @@
             -maxCharge $advanced.maxCharge
             -n $advanced.n
             -addFeatures $advanced.addFeatures
+	#if $tsvcheck
+	    &&
+	    msgf_plus
+	        edu.ucsd.msjava.ui.MzIDToTsv
+	        -i "$output_name"
```

Comment:
No, nothing to do with Python. The reational behind it is, that on bash the `"` are interpreted. So assuming someone bad includes some strange characters in your text-param or in the file-system path via filenames or what ever, bash will try to interpret it. Single quotes don't do this and hence are more safe.
This should work for everything in cheetah, it's just a bashism-